addons/crm/static/src/legacy/models/activity_group_view.js

```diff
@@ -1,35 +0,0 @@
-/** @odoo-module **/
-
-import { Patch } from '@mail/legacy/model';
-
-Patch({
-    name: 'ActivityGroupView',
-    recordMethods: {
-        /**
-         * @override
-         */
-        onClickFilterButton(ev) {
```

Comment:
This is currently being removed without being converted.

Reply:
Structurally converted. Haven't checked yet whether functionally working as expected